var/spack/repos/builtin/packages/py-bokeh/package.py

```diff
@@ -48,3 +58,5 @@ class PyBokeh(PythonPackage):
 
     depends_on("py-typing-extensions@3.7.4:", type=("build", "run"), when="@2.3.3:")
     depends_on("py-typing-extensions@3.10.0:", type=("build", "run"), when="@2.4.0:")
```

Comment:
typing-extensions is no longer required

Reply:
```suggestion
    depends_on("py-typing-extensions@3.7.4:", type=("build", "run"), when="@2.3.3:3.0.0")
    depends_on("py-typing-extensions@3.10.0:", type=("build", "run"), when="@2.4.0:3.0.0")
```

Reply:
^ Note: yes, 3.0.0 inclusive. Typing extension dependency removed in 3.0.1.